### PR TITLE
Add a nev bar

### PR DESCRIPTION
**Pull request by aslihanasadov**
NavBar completed and router issue has been fixed

### CODE REVIEW

**Comment by aslihanasadov** [comment]
changes made in PetTypesContainer.

File: src/components/App.js [hunk 1]
```diff
@@ -1,25 +1,15 @@
...
-          <Route exact path="/pets/:animalType/:id" component={PetShowContainer} />
-          <Route exact path="/pets/:type" component={PetTypesContainer} />
-          <Route path="*" component={NotFound} />
-        </Switch>
-      </BrowserRouter>
-      <Footer />
-    </Fragment>
-  )
-}
+    <div>
```
Comment (DanAlberts) [inline src/components/App.js]: These lines need to be indented properly

File: src/components/ListPets.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useState } from 'react';
 import Pet from "./Pet"
+import NavBar from './NavBar';
```
Comment (DanAlberts) [inline src/components/ListPets.js]: The NavBar component does not need to be imported here if it isn't being used or rendered in this component.

File: src/components/NavBar.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                            <Link to="/pets">HOME</Link>
+                        </li>
+                        <li>
+                            <Link to="/pets/dog">DOGS</Link>
+                        </li>
+                        <li>
+                            <Link to="/pets/cat">CATS</Link>
+                        </li>
+                    </ul>
+            </nav>
```
Comment (DanAlberts) [inline src/components/NavBar.js]: the </nav tag and switch tags need to be indented properly.

File: src/components/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            <Switch>
+                <Route exact path="/pets" component={ListPets} />
+                <Route exact path="/pets/:type" component={PetTypesContainer} />
+                <Route exact path="/pets/:animalType/:id" component={PetShowContainer} />
+                <Route path="*" component={NotFound} />
+            </Switch>
+            </div>
+        </BrowserRouter>
+
```
Comment (DanAlberts) [inline src/components/NavBar.js]: There is no need for this empty line of space on line 35

File: src/components/Pet.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ const Pet = props => {
         <div className="small-6 columns">
           <div className="card-animal">
             <div class="card-animal-body">
-              <Link to="/pets">
+            <Link to={`/pets/${props.type}`}>
```
Comment (DanAlberts) [inline src/components/Pet.js]: This needs to be indented properly

**Pull request merged**